.codegen/service.go.tmpl

```diff
@@ -109,16 +109,19 @@ var {{.CamelName}}Overrides []func(
 	{{- end }}
 )
 
+{{- $excludeFromJson := list "http-request"}}
```

Comment:
This is a workaround to avoid conflicting flags coming from "serving-endpoints.http-request" and global json we provide. We need to find a better way to deal with it. cc @pietern @denik

Reply:
Thanks -- this seems like a call we would need to special-case anyway.